src/Vogen.Pack/Vogen.Pack.csproj

```diff
@@ -57,13 +57,27 @@
-
     <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.8\$(Configuration)\netstandard2.0\Vogen.SharedTypes.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
-
     <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.8\$(Configuration)\netstandard2.0\Vogen.SharedTypes.dll" Pack="true" PackagePath="lib\netstandard2.0" Visible="true" />
     <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.8\$(Configuration)\net9.0\Vogen.SharedTypes.dll" Pack="true" PackagePath="lib\net9.0" Visible="true" />
     <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.8\$(Configuration)\net9.0\Vogen.SharedTypes.xml" Pack="true" PackagePath="lib\net9.0" Visible="true" />
     <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.8\$(Configuration)\net9.0\Vogen.SharedTypes.xml" Pack="true" PackagePath="lib\netstandard2.0" Visible="true" />
+
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.11\$(Configuration)\netstandard2.0\Vogen.dll" Pack="true" PackagePath="analyzers/dotnet/roslyn4.11/cs/" Visible="false" />
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen.CodeFixers\bin\roslyn4.11\$(Configuration)\netstandard2.0\Vogen.CodeFixers.dll" Pack="true" PackagePath="analyzers/dotnet/roslyn4.11/cs" Visible="false" />
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.11\$(Configuration)\netstandard2.0\Vogen.SharedTypes.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.11\$(Configuration)\netstandard2.0\Vogen.SharedTypes.dll" Pack="true" PackagePath="lib\netstandard2.0" Visible="true" />
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.11\$(Configuration)\net9.0\Vogen.SharedTypes.dll" Pack="true" PackagePath="lib\net9.0" Visible="true" />
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.11\$(Configuration)\net9.0\Vogen.SharedTypes.xml" Pack="true" PackagePath="lib\net9.0" Visible="true" />
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.11\$(Configuration)\net9.0\Vogen.SharedTypes.xml" Pack="true" PackagePath="lib\netstandard2.0" Visible="true" />
+
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.12\$(Configuration)\netstandard2.0\Vogen.dll" Pack="true" PackagePath="analyzers/dotnet/roslyn4.12/cs/" Visible="false" />
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen.CodeFixers\bin\roslyn4.12\$(Configuration)\netstandard2.0\Vogen.CodeFixers.dll" Pack="true" PackagePath="analyzers/dotnet/roslyn4.12/cs" Visible="false" />
+    <None Include="$(MSBuildThisFileDirectory)\..\Vogen\bin\roslyn4.12\$(Configuration)\netstandard2.0\Vogen.SharedTypes.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```

Comment:
I think it's these copies that are causing the build issue:

`error NU5118: Warning As Error: File '/Users/runner/work/Vogen/Vogen/src/Vogen/bin/roslyn4.11/Debug/netstandard2.0/Vogen.SharedTypes.dll' is not added because the package already contains file 'analyzers/dotnet/cs/Vogen.SharedTypes.dll' [/Users/runner/work/Vogen/Vogen/src/Vogen.Pack/Vogen.Pack.csproj]`

It looks like the output from the most recent roslyn build is copied over